sdk/eventgrid/Azure.Messaging.EventGrid/src/Customization/SystemEventNames.cs

```diff
@@ -32,7 +32,7 @@ public static class SystemEventNames
         /// The value of the Event Type stored in <see cref="EventGridEvent.EventType"/> and <see cref="CloudEvent.Type"/> for the
         /// <see cref="AcsChatParticipantAddedToThreadEventData"/> system event.
         /// </summary>
-        public const string AcsChatParticipantAddedToThread = "Microsoft.Communication.ChatParticipantAddedToThread";
+        public const string AcsChatParticipantAddedToThread = "Microsoft.Communication.ChatThreadParticipantAdded";
```

Comment:
Feels like we need to add something for `Microsoft.Communication.RecordingFileStatusUpdated` to this file?

Reply:
Yep, looks like the test actually works! https://dev.azure.com/azure-sdk/public/_build/results?buildId=796869&view=ms.vss-test-web.build-test-results-tab&runId=17558288&resultId=100009&paneView=debug